aws/default_retryer.go

```diff
@@ -34,25 +35,39 @@ var seededRand = rand.New(&lockedSource{src: rand.NewSource(time.Now().UnixNano(
 // RetryRules returns the delay duration before retrying this request again
 func (d DefaultRetryer) RetryRules(r *Request) time.Duration {
 	// Set the upper limit of delay in retrying at ~five minutes
-	minTime := 30
+	var minTime int64
+	if d.MinTime != 0 {
+		minTime = d.MinTime
+	} else {
+		minTime = 30
```

Comment:
Since the min retry and throttle delay will be configuration should create constants for both that document the default minimum delay.

```go
const (
    DefaultRetryerRetryDelay    = 30 * time.Milliseconds 
    DefaultRetryerThrottleDelay = 30 * time.Milliseconds
    DefaultRetryerMaxNumRetries = 3
)
```